src/bitwarden_exporter/__main__.py

```
@@ -27,6 +27,65 @@
+            if len(jmespath_password) == 0:
+                raise BitwardenException("JMESPath expression returned empty list")
+            jmespath_password = jmespath_password[0]
+
+        if not isinstance(jmespath_password, str):
+            raise BitwardenException("Vault password is not a string")
+
+        LOGGER.warning("Vault password is set from JMESPath expression")
+        secret_path = jmespath_password
+
+    if os.path.exists(secret_path) and os.path.isfile(secret_path):
+        with open(secret_path, "r", encoding="utf-8") as f:
```

Ruff (0.14.1)

Unnecessary mode argument

Remove mode argument

(UP015)